src/docs/app/shared/component-viewer/component-viewer/component-viewer.component.scss

```diff
@@ -0,0 +1,13 @@
+:host {
+  display: block;
+  padding: 1.5rem;
+}
+
+nav {
+  display: flex;
+  gap: var(--sbb-spacing-fixed-6x);
+}
+
+sbb-link-button {
+  font-size: var(--sbb-font-size-text-xl);
```

Comment:
I don't feel we should overwrite lyne components. if really needed, letter-spacing should be added as well. BUT, we have CSS classes for font-sizes: in this case it would be "sbb-text-xl"